uwsm-plugins/meson.build

```diff
@@ -0,0 +1,11 @@
+uwsm_plugins = [
+  'labwc.sh',
+  'wayfire.sh',
+  'sway.sh',
+  'Hyprland.sh',
+]
+
+install_data(
+  uwsm_plugins,
+  install_dir: PKG_DATA_DIR / 'plugins',
```

Comment:
Why move plugins away from lib?

Reply:
`/usr/share` seems to be the preferred location for sourced scripts.
There's a handful of packages that use `/usr/lib`, so it doesn't seem to be a hard rule.

e.g
bash: `/usr/share/bash-completion`
zsh : `/usr/share/zsh/functions`
nano: `/usr/share/nano/*.nanorc`
mpv: `/usr/share/mpv/scripts/*.sh`
etc

obs-studio is an interesting one. Because their plugins are in C/C++, the shared objects are stored in `/usr/lib/obs-plugins/`, but their runtime data is put in `/usr/share/obs/obs-plugins/<plugin-name>/`.
e.g the (3rd party) obs-vkcapture plugin:
```
/usr/lib/obs-plugins/linux-vkcapture.so
/usr/share/obs/obs-plugins/linux-vkcapture/
/usr/share/obs/obs-plugins/linux-vkcapture/locale/
/usr/share/obs/obs-plugins/linux-vkcapture/locale/cs-CZ.ini
/usr/share/obs/obs-plugins/linux-vkcapture/locale/de-DE.ini
/usr/share/obs/obs-plugins/linux-vkcapture/locale/en-US.ini
/usr/share/obs/obs-plugins/linux-vkcapture/locale/es-ES.ini
/usr/share/obs/obs-plugins/linux-vkcapture/locale/fr-FR.ini
/usr/share/obs/obs-plugins/linux-vkcapture/locale/it-IT.ini
/usr/share/obs/obs-plugins/linux-vkcapture/locale/ja-JP.ini
/usr/share/obs/obs-plugins/linux-vkcapture/locale/pl-PL.ini
/usr/share/obs/obs-plugins/linux-vkcapture/locale/pt-BR.ini
/usr/share/obs/obs-plugins/linux-vkcapture/locale/ru-RU.ini
/usr/share/obs/obs-plugins/linux-vkcapture/locale/uk-UA.ini
```

More on the topic: https://bbs.archlinux.org/viewtopic.php?pid=2049023